web/satellite/src/types/payments.ts

```diff
@@ -69,3 +77,43 @@ export class PaymentAmountOption {
         public label: string = '',
     ) {}
 }
+
+// BillingHistoryItem holds all public information about billing history line.
+export class BillingHistoryItem {
+    public constructor(
+        public readonly id: string = '',
+        public readonly description: string = '',
+        public readonly amount: number = 0,
+        public readonly status: string = '',
+        public readonly link: string = '',
+        public readonly start: Date = new Date(),
```

Comment:
shouldn't name of the variable be more clear? f.e. startDate?
same with end

Reply:
```start: Date``` is easy to understand :D
will ```startDate: Date``` be better or it will be just a redundant suffix?